pkg/app/up.go

```diff
@@ -90,6 +90,16 @@ var CommandUp = &cli.Command{
 			Usage: "Configure the shared memory size (megabyte)",
 			Value: 2048,
 		},
+		&cli.StringFlag{
```

Comment:
should we add another flag named `enable-cpu-binding` to make people can decide to bind CPU core nor not

Reply:
Do you mean `--cpuset-cpus` for docker?

Reply:
> Do you mean `--cpuset-cpus` for docker?

Yes, the `cpuset-cpus` means binding the workload with CPU core to reach the max performance